module/Database/src/Model/Member.php

```diff
@@ -510,8 +538,12 @@ public function getBirth(): DateTime
     /**
      * Set the birthdate.
      */
-    public function setBirth(DateTime $birth): void
+    public function setBirth(DateTime|string $birth): void
     {
+        if (is_string($birth)) {
+            $birth = new DateTime($birth);
+        }
```

Comment:
Did this come from static analysis? Imho, there should be no reason to allow `string` here and do a conversion. The provided data should be `DateTime` before passing it on.

Reply:
Yes